### ISSUE TITLE
Make link color darker

### ISSUE THREAD

**Issue by asahiko** [antleaf/nims-hyrax#205]
We've been following the [appearance settings recommendation in this comment](https://github.com/antleaf/nims-mdr-development/issues/1#issuecomment-500437970), but the link color `#AFCA0B` is too bright, making it hard to read on white backgrounds. The contrast does not meet the Web Content Accessibility Guidelines. Contrast checker tool: https://webaim.org/resources/contrastchecker/?fcolor=AFCA0B&bcolor=FFFFFF

![mdr-green](https://user-images.githubusercontent.com/689506/80075255-63384c80-8585-11ea-9282-90466810a740.png)

[The designer actually specified](https://github.com/antleaf/nims-mdr-development/blob/master/Design/Branding%20Design/MDR%20Branding%20%26%20Digital%20Style/mdr%20digital%20styling/MDR%20Digital%20Styling%20%5Bphase%204%5D.pdf) a slightly darker shade of green for white backgrounds, `#7DB928`, but this color also does not meet the guidelines. https://webaim.org/resources/contrastchecker/?fcolor=7DB928&bcolor=FFFFFF

The color for when the link is inactive can be changed in the admin dashboard without changes to the stylesheet, but that setting only overrides inactive `a`, while leaving `a:focus, a:hover, a:active` at the old `$mdr-green` i.e. `#AFCA0B`. Color highlight on mouse hover is a thing, but the change is too steep, and it doesn't match how the buttons don't respond to mouse hover.

![mouse-over](https://user-images.githubusercontent.com/689506/80076225-d5f5f780-8586-11ea-9dbe-d47ce464f2f5.png)

Darkening the designer's `#7DB928` to an acceptable contrast level will put us in the area around `#51791A`. https://webaim.org/resources/contrastchecker/?fcolor=51791A&bcolor=FFFFFF
